sceptre/cli/helpers.py

```diff
@@ -173,6 +189,23 @@ def _generate_text(stream):
     return stream
 
 
+def process_template(
+    template: dict, output_format: str, type: str = "template"
```

Comment:
It's generally discouraged to shadow builtins like `type`.

Reply:
fixed